frontend/pages/queries/ManageQueriesPage/components/QueriesTable/QueriesTable.tsx

```diff
@@ -118,20 +118,20 @@ const QueriesTable = ({
                 .toLowerCase()
                 .includes(queryParams?.query.toLowerCase())
             : true;
-
           const compatiblePlatforms =
             checkPlatformCompatibility(query.query).platforms || [];
 
-          const filterCompatiblePlatform = queryParams?.platform
-            ? compatiblePlatforms.includes(queryParams?.platform)
-            : true;
+          const filterCompatiblePlatform =
+            queryParams?.platform && queryParams?.platform !== "all"
+              ? compatiblePlatforms.includes(queryParams?.platform)
+              : true;
 
           return filterSearchQuery && filterCompatiblePlatform;
         }) || []
       );
     }
     setIsQueriesStateLoading(false);
-  }, [queriesList, queryParams?.query]);
+  }, [queriesList, queryParams]);
```

Comment:
Corrected to update if any changes to queryParam so the list/counts are updated when modifying platforms